chromium_src/courgette/third_party/bsdiff/paged_array.h

```diff
@@ -0,0 +1,31 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_COURGETTE_THIRD_PARTY_BSDIFF_PAGED_ARRAY_H_
+#define BRAVE_CHROMIUM_SRC_COURGETTE_THIRD_PARTY_BSDIFF_PAGED_ARRAY_H_
+
+#define PagedArray PagedArray_ChromiumImpl
+#define kPagedArrayDefaultPageLogSize kPagedArrayDefaultPageLogSize_ChromiumImpl
+
+#include "src/courgette/third_party/bsdiff/paged_array.h"
+
+#undef PagedArray
+#undef kPagedArrayDefaultPageLogSize
```

Comment:
nit: undef order should be reversed.

Reply:
I've reversed it. Thank you for your feedback!